docs/orchestration/concepts/api.md

```diff
@@ -63,28 +59,24 @@ client.graphql(
 
 ### Authenticating the Client with Cloud <Badge text="Cloud"/>
 
-If using Prefect Cloud, authentication is required. For interactive use, the most common way to use the Cloud Client is to generate a `USER`-scoped token and provide it to the client. After doing so, users can save the token so it persists across all Python sessions:
+If using Prefect Cloud, authentication is required. For interactive use, the most common way to use the Cloud Client is to generate a user-based API key and provide it to the client. After doing so, users can save the key so it persists across all Python sessions:
 
 ```python
 import prefect
-client = prefect.Client(api_token="YOUR_USER_TOKEN")
+client = prefect.Client(api_token="YOUR_USER_KEY")
 client.save_api_token()
 ```
 
 Now, starting a client in another session will automatically reload the token:
 
 ```python
 client = prefect.Client()
-assert client._api_token == "YOUR_USER_TOKEN"  # True
+assert client._api_token == "YOUR_USER_KEY"  # True
 ```
 
-Note that a token can be provided by environment variable (`PREFECT__CLOUD__AUTH_TOKEN`) or in your Prefect config (under `cloud.auth_token`).
-
-:::tip Using `USER` tokens
-The steps shown here were designed to be used with `USER`-scoped tokens. They will work with `TENANT` scoped tokens as well, but unexpected errors could result.
-:::
+Note that a key can be provided by environment variable (`PREFECT__CLOUD__AUTH_TOKEN`) or in your Prefect config (under `cloud.auth_token`).
 
-Once provisioned with a `USER` token, the Cloud Client can query for available tenants and login to those tenants. In order to query for tenants, call:
+Once provisioned with a user-based API key, the Cloud Client can query for available tenants and login to those tenants. In order to query for tenants, call:
```

Comment:
```suggestion
Once provisioned with an API key, the Cloud Client can query for available tenants and login to those tenants. In order to query for tenants, call:
```

Reply:
same question here re: user vs service account API key, that's definitely why I worded it this way initially